### PR TITLE
Update icons to latest release

### LINKED ISSUE #465: Update to the newest SVG icons
# Issue Description

Update to the newest SVGs in cerner-one-style-icons. There have been changes since the initial 1.0 release.
https://github.com/cerner/terra-core/blob/master/packages/terra-icon/package.json#L25

https://github.com/cerner/one-cerner-style-icons/commits/master

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Resolves #465. Most of this code change is the result of our automated build tasks. I updated the version of icons in the package.json and then ran `npm run migrate-cerner-one-icons` to generate all of these updates.

#### Added
 * Added compose icon.
 * Added microphone icon.
 * Added microphoneDisabled icon.
 * Added phoneDown icon.
 * Added question icon.
 * Added swap icon.
 * Added unexpected icon.
 * Added unread icon.
 * Added unspecified icon.
 * Added videoCameraDisabled icon.
 
 #### Changed
 * Changed path data for unavailable icon.

### Deployment Link
https://terra-core-deployed-pr-762.herokuapp.com/static/#/site/icon

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

**Comment by mjhenkes** [comment]
1.4.1 was just released.
https://github.com/cerner/one-cerner-style-icons/tree/1.4.1

**Comment by bjankord** [comment]
@mjhenkes Updated to latest OCS icon set.

File: packages/terra-icon/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   "devDependencies": {
     "csvtojson": "^1.1.4",
     "jsdom": "^9.11.0",
-    "one-cerner-style-icons": "https://github.com/cerner/one-cerner-style-icons/archive/v1.0.0.tar.gz",
+    "one-cerner-style-icons": "https://github.com/cerner/one-cerner-style-icons/archive/v1.4.2.tar.gz",
```
Comment (mjhenkes) [inline packages/terra-icon/package.json]: they re-released a 1.4.2 version without the v

Comment (mjhenkes) [reply packages/terra-icon/package.json]: https://github.com/cerner/one-cerner-style-icons/tree/1.4.2

**Comment by mmalaker** [comment]
Icon examples look good with the exception of the Help icon in IE. There is a strange, ellipse-like artifact displaying on top of the icon.

**Comment by bjankord** [comment]
Removing the "Unexpected" icon seems to remove the rendering artifact in IE. I'll follow up with UX and see if there is clean up they need to do on the icon file.

screen shot 2017-09-03 at 7 27 10 am

**Comment by bjankord** [comment]
There are extra `<rect />` elements in one of the icons. IE seems to be rendering these extra SVG elements. Removing these should remove the squares while the actual icon mark remains the same. Logged an issue to the UX icon repo to see if we can get a fix for it. https://github.com/cerner/one-cerner-style-icons/issues/15

**Comment by bjankord** [comment]
@mmalaker The extra markings appearing in IE should be removed now. 
screen shot 2017-09-06 at 11 36 09 am

**Comment by mmalaker** [comment]
Functional verification completed. New icon examples display correctly in IE and other supported browsers.

**Pull request merged**